README.md

```diff
@@ -82,3 +83,7 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
 - [nova-chartjs](https://github.com/coroo/nova-chartjs) - Laravel Nova
 - [quickchart](https://github.com/typpo/quickchart) - Web API for static charts
 - [wicked-charts](https://github.com/adessoAG/wicked-charts) - Java
+
+
+### Other
+ - [xdoc](https://github.com/nschloe/xdoc) - Browser extension for using Chart.js on GitHub
```

Comment:
`nschloe/xdoc` redirects to `nschloe/xhub` where there is no mention of "xdoc". Should rename the link to `xhub`.